02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,110 @@
+      </p>
+      <p>
+        I have a wonderful family with a son in his last semester in college and a wife that is an awrd winning playwright and poet.  These two are the reason I have always worked so hard, they are my center and my reason to live.  Now that school is coming to a close for my son and my wfe is getting more established time with them is become more of a reality and I want to spend time with them and have many years of cherished memories.
+      </p>
+      <p>
+        Exercise is a way for me to reduce stress and live a longer life that will continue to have a quality of life so I can enjoy my family, have fun and continue to be productive in my work.  I enjoy most sports but I really enjoy walking.  My wife and I walk as often as we can and have a goal to walk daily.  Our typical walks are 5 miles long and are members of AVA which is a walking club.  Other than walking I like to swim and thoroughly enjoy SCUBA diving.  I am a certified diver and look forward to another opportunity to dive.  I like to play golf although it is difficult to call my version actual golf.
+      </p>
+      <p>
+        Fun involves several things for me.  I enjoy traveling and I have been many places. There are numerous places that I still want to visit before I die.  Some of the places I have visited include; 45 of the 50 States in the United States, Russia, Japan, Samoa, Panama, Peru, Mexico, Canada, Dominican Republic, Belize, Ireland and most recently I was fortunate enough to be able to spend a week in Paris, France.  Other than traveling SCUBA diving is not only exercise but a lot of fun along with hiking and site seeing.  Being with my family is what I enjoy and being with them and enjoying life is the best of all worlds.
+      </p>
+      <p>
+        As mentioned work is still important to me and being able to be a productive member of society is crucial.  I will still work for many years and all I ask from an employer is to be paid what I am worth.  Typically, most companies pay their employees the lowest wages they can get away with and still keep their employees. I have heard there are companies, however, who pay their employees what they are worth as an incentive to keep them. I have not yet found such a company but I am still hopeful.
+      </p>
+      <p>
+        In closing let me say that I am a blessed person that has a loving family, good health and a job that currently pays my bills.  I am looking for that perfect company that will afford me the ability to have a balance and harmony in my life that I am looking for and allow me to travel more with my wife either for work or for pleaseure or both.
+      </p>
+    </section>
+
+    <aside>
+      <ul type="circle">Social Media I am on:
```

Comment:
great idea to put these in! make the links go directly to your page, if you would.